src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
-'use strict';
+const succsesFunc = (message) => {
+  const successDiv = document.createElement('div');
 
-// write your code here
+  successDiv.className = 'message';
```

Comment:
The class name 'message' is quite generic. If this is intended for success messages, consider a more specific class name such as 'success-message' to distinguish it from error messages and other types of messages.